### PR TITLE
addpkg: v2ray-desktop

### PR DESCRIPTION
**Pull request by Dr-Incognito**
Create package for V2Ray-Desktop.

AUR: https://aur.archlinux.org/packages/v2ray-desktop/
GitHub: https://github.com/Dr-Incognito/V2Ray-Desktop

### CODE REVIEW

File: archlinuxcn/v2ray-desktop/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,49 @@
+# Maintainer: Dr-Incognito <v2ray.dev at gmail dot com>
+pkgname=v2ray-desktop
+pkgver=1.5.1
+pkgrel=1
+pkgdesc="A Cross-platform GUI Client for V2Ray written in Qt 5 and QML 2 (Stable Release)"
+arch=("x86_64")
+url="https://github.com/Dr-Incognito/V2Ray-Desktop"
+license=("GPL3")
+depends=("v2ray" "qt5-base>=5.14.0" "qt5-quickcontrols>=5.14.0" "qt5-quickcontrols2>=5.14.0")
+makedepends=("git>=2.11" "qt5-tools>=5.14.0" "qt5-declarative>=5.14.0" curl which gcc make jq sed)
```
Comment (imlonghao) [inline archlinuxcn/v2ray-desktop/PKGBUILD]: > 在使用makepkg构建软件包时，base-devel组被视为已安装，该组的成员不应该出现在makedepends列表中

`sed` `make` `gcc` `which` 属于 `base-devel` 组

Comment (Dr-Incognito) [reply archlinuxcn/v2ray-desktop/PKGBUILD]: Updated. Thank you for reviewing this PR.

**Comment by Dr-Incognito** [comment]
@imlonghao 
Please review this PR again.
Thank you.

**Comment by imlonghao** [comment]
Any plan for https://github.com/Dr-Incognito/V2Ray-Desktop/issues/2 ?

**Comment by Dr-Incognito** [comment]
OK. I'll fix it ASAP.

**Comment by Dr-Incognito** [comment]
It seems caused by conflicts of QML and KDE.

File: archlinuxcn/v2ray-desktop/lilac.yaml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+maintainers:
+  - github: Dr-Incognito
```
Comment (imlonghao) [inline archlinuxcn/v2ray-desktop/lilac.yaml:2]: You don't have a public email on your github profile page, so you need to specify your email address here.

File: archlinuxcn/v2ray-desktop/PKGBUILD [hunk 3]
```diff
@@ -0,0 +1,41 @@
+# Maintainer: Dr-Incognito <v2ray.dev at gmail dot com>
+pkgname=v2ray-desktop
+pkgver=2.0.0
+pkgrel=1
+pkgdesc="A cross-platform GUI proxy client that supports Shadowsocks, V2Ray, and Trojan protocols (Stable Release)"
+arch=("x86_64")
+url="https://github.com/Dr-Incognito/V2Ray-Desktop"
+license=("GPL3")
+depends=("clash" "qt5-base>=5.14.0" "qt5-quickcontrols>=5.14.0" "qt5-quickcontrols2>=5.14.0")
+makedepends=("git>=2.11" "qt5-tools>=5.14.0" "qt5-declarative>=5.14.0")
```
[the remark this answers is not in the document]
Comment (Dr-Incognito) [reply archlinuxcn/v2ray-desktop/PKGBUILD:10]: Thank you very much!

**Pull request merged**